pyproject.toml

```diff
@@ -1,3 +1,37 @@
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.mypy]
+python_version = "3.10"
+files = [
+  "result",
+  "tests",
+]
+# Mypy Python 3.10 support: https://github.com/python/mypy/issues/10201
```

Comment:
This comment may be easily midunderstood. Perhaps,
> Need to exclude pattern matching until Mypy Python 3.10 support is ready, https://github.com/python/mypy/issues/10201

?

Reply:
i agree, turned it into

```
# Exclude files with pattern matching syntax until Mypy supports it;
# see https://github.com/python/mypy/issues/10201
```